index.js

```diff
@@ -0,0 +1,69 @@
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+const bookContainer = document.querySelector('#ptext');
+const addBtn = document.getElementById('add-btn');
+
+let books = [];
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  books = JSON.parse(localStorage.getItem('books'));
+}
```

Comment:
- [OPTIONAL] We can refactor this to one line of code like this 👇🏼 using the OR(||) operator.

```suggestion
let books = JSON.parse(localStorage.getItem('books')) || [];
```